code/modules/recycling/conveyor2.dm

```diff
@@ -387,6 +354,42 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	transfer_fingerprints_to(NC)
 	qdel(src)
 
+/obj/item/stack/conveyor
+	name = "conveyor belt assembly"
+	desc = "A conveyor belt assembly."
+	icon = 'icons/obj/recycling.dmi'
+	icon_state = "conveyor_construct"
+	max_amount = 30
+	singular_name = "conveyor belt"
+	///id for linking
+	var/id = ""
+
```

Comment:
<2/3 stacks now fit in a box, and maxed stacks are only normal so fit in backpack. Should add a new min_w_class to stacks as well as override full_w_class then make it bulky. Otherwise, at least add to changelog.